test/unit/lib/plugins/aws/deploy/lib/extendedValidate.test.js

```diff
@@ -90,27 +90,25 @@ describe('extendedValidate', () => {
       expect(() => awsDeploy.extendedValidate()).to.throw(Error);
     });
 
-    it('should not throw error if service has no functions and no service package', () => {
+    it('should not throw error if service has no functions and no service package', async () => {
       stateFileMock.service.functions = {};
       fileExistsSyncStub.returns(true);
       readFileSyncStub.returns(stateFileMock);
 
-      return awsDeploy.extendedValidate().then(() => {
-        expect(fileExistsSyncStub.calledOnce).to.equal(true);
-        expect(readFileSyncStub.calledOnce).to.equal(true);
-      });
+      await awsDeploy.extendedValidate();
```

Comment:
Does it have to be `await`-ed here? According to the changes to `extendedValidate`, that function is no longer async

Reply:
Ah, good catch. I'll fix all of those `await`'s.